src/main/java/com/esri/samples/scene/add_a_point_scene_layer/README.md

```diff
@@ -0,0 +1,41 @@
+<h1>Add a point scene layer</h1>
+
+<p>View a point scene layer from a scene service.</p>
+
+<p><img src="AddAPointSceneLayer.png"/></p>
+
+<h2>Use case</h2>
+
+<p>Point scene layers can efficiently display large amounts of point features. While point cloud layers can only display simple symbols, point scene layers can display any type of billboard symbol or even 3D models, as long as the location of the symbol can be described by a point. Points are cached and automatically thinned when zoomed out to improve performance.</p>
```

Comment:
To really emphasise this point...could each point be represented by our 3D plane model instead of a simple white circle? It would mean having to change the base data in Pro/AGOL though. Just a thought.

Reply:
I think this is a good idea but not sure we have control over the service. We can look at it again after the release.